client/src/main/java/com/flipkart/flux/client/runtime/FluxRuntimeConnectorHttpImpl.java

```diff
@@ -51,6 +51,7 @@
     public static final int MAX_TOTAL = 200;
     public static final int MAX_PER_ROUTE = 20;
     public static final String EXTERNAL = "external";
+    public static final String EXTERNAL_UPDATE = "externalUpdate";
```

Comment:
Why do we need this attribute if we have a specific api for the update purpose..

Reply:
This attribute is used as a default value for "eventSource" column in updated Event DB schema, if user passes null. While updating  eventData via 
    public Event persistEvent(String stateMachineInstanceId, EventData eventData) {
.
event.setEventData(eventData.getData());
event.setEventSource(eventData.getEventSource());
.
}
"eventSource" is required in "eventData" object. And it gives us a separation of all list of events updated externally, filtered on "externalUpdate" eventSource.

Reply:
user can just pass "external" as the source, as is the case with external events, we don't need this extra attribute, as the update is via specific api..

Reply:
Added it to differentiate eventUpdates. Will remove and use default existing value "external" itself to refrain from adding additional attributes.